src/allmydata/interfaces.py

```diff
@@ -351,6 +351,10 @@ def slot_testv_and_readv_and_writev(
     ):
         """
         :see: ``RIStorageServer.slot_testv_readv_and_writev``
+
+        While the interface mostly matches, test vectors are simplified.
+        Instead of a tuple ``(start, offset, operator, data)`` they are just
```

Comment:
I think the tuple from the interface method is "(offset, read_length, operator, specimen)" (esp. the first two elements).